### ISSUE TITLE
Build Web Apps with Express.js | STATIC | missed path and passed in the verification

### ISSUE THREAD

**Issue by FabioRam** [freeCodeCamp/freeCodeCamp#15527]
#### Challenge Name
Build Web Apps with Express.js | STATIC

#### Issue Description
- The issue is regarding the exercise STATIC. My code (below) passed in the verification, but I didn't insert ```var path = require('path');``` the requirement of 'pass' although I used it following the suggestion in the Hint.

NOTE: In the Header of the page https://www.freecodecamp.com/challenges/build-web-apps-with-expressjs, it is "Expressjs" instead of "Express.js".

#### Your Code

```js
var express = require('express');
var app = express();

app.use(express.static(process.argv[3]||path.join(__dirname, 'public')));

app.listen(process.argv[2]);


```

**Comment by MiloATH** [comment]
Hi @FabioRam, 

Thanks for reporting this issue.

Unfortunately, that challenge isn't managed by FCC. [Here is the repo of expressworks.](https://github.com/azat-co/expressworks) This issue is better suited on that repo.

Happy Coding!

**Comment by MiloATH** [comment]
closes #15527

**Comment by erictleung** [comment]
@FabioRam thanks for the issue. As @MiloATH has pointed out, this part of the curriculum is not ours. Please open an issue over at https://github.com/azat-co/expressworks/issues to get it fixed. And thanks for the note. I think the head is dynamically generated and there may be an issue with parsing periods. Nonetheless, closing as an issue we don't have full control over. Happy coding!
